card/lib/card/content.rb

```diff
@@ -179,6 +179,12 @@ def clean!( string, tags = ALLOWED_TAGS )
         end.gsub(/<\!--.*?-->/, '')
       end
 
+      if Card.config.space_last_in_multispace
```

Comment:
It seemed best to have this happen at the same time as tag cleaning.  It doesn't make sense except when you have entity encoding, which is generally HTML contexts where we do tag cleaning.